rpcs3/Emu/RSX/GL/rsx_gl_texture.cpp

```diff
@@ -138,7 +137,22 @@ namespace rsx
 				glPixelStorei(GL_UNPACK_SWAP_BYTES, GL_TRUE);
 				glPixelStorei(GL_UNPACK_ROW_LENGTH, pitch / 2);
 
-				// TODO: texture swizzling
+				if (is_swizzled)
+				{
+					u16 height = tex.height();
```

Comment:
@paulsapps I think as vlj already said, it should probably use the TextureUtils, as the code for that should be there already.